uniflow/op/model/model_config.py

```diff
@@ -61,3 +61,15 @@ class NougatModelConfig(ModelConfig):
     model_name: str = "0.1.0-small"
     batch_size: int = 1
     model_server: str = "NougatModelServer"
+
+@dataclass
+class BedrockModelConfig(ModelConfig):
+    """Bedrock Model Config Class."""
+
+    model_name: str = "anthropic.claude-v2"
+    batch_size: int = 1
```

Comment:
qq: does Bedrock Claude endpoint supports `batch inference` or `num_call` like OpenAI endpoint?

Reply:
Currently, the Bedrock Claude endpoint supports batch inference by submitting an asynchronous inference job. The mechanism might not be exactly the same as the OpenAI endpoint. Ref: https://docs.aws.amazon.com/bedrock/latest/userguide/batch-inference-create.html

Reply:
i c. OpenAI endpoint does not support `batch inference` but a `num_call` to invoke the API num_call times. Regarding the RedRock batch inference async API. does it allow to configure the batch size? I do not see it in the doc above. It looks to me this batch inference is to async process a list of prompt instead of running a batch of data in parallel. Is my understanding correct?

Reply:
I think it depends on the implementation from Bedrock. I agree with your opinion that this async API might resemble a 'fake' batch inference, or it could be a case where Bedrock combines the JSONL (possibly with other requests) to help users manage the batch. We can dive deeper into the mechanism of Bedrock in the future to clarify this.